src/lib.rs

```diff
@@ -46,7 +47,7 @@
 //! ```
 //!
 //! Check out [the examples on GitHub](https://github.com/LukeMathWalker/tracing-actix-web/tree/main/examples) to get a taste of how [`TracingLogger`] can be used to observe and monitor your
-//! application.  
+//! application.
```

Comment:
Double whitespace at the end is significant here—please restore the README to what it used to be, minus the additional feature flag.

Reply:
My IDE erased them automatically.
May I know why are they important? Because it's better not to be dependent to something that nobody can see :D

Reply:
I manually removed them from `Readme.md` in master, and it looked the same.

Reply:
Double-spaces at the end of a line in a Markdown document stand for a new line.

Reply:
Oh, didn't know that br thingy in md. btw, I don't like it :D
But it's OK, I reverted them back.

Some of them are still useless I guess, because they have a new line, after them. Like the one you put a comment on.
in the doc it will be:
```
<p>x<br></p>
```